.github/composite-actions/build-pg-hint-plan/action.yml

```diff
@@ -0,0 +1,14 @@
+name: 'Build pg-hint-plan'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout, Build and Install pg-hint-plan
```

Comment:
Since use of pg_hint_plan is optional, I think we also need a test run that does NOT include it.

Reply:
I have added a separate flow for pg_hint_plan instead of merging it with the JDBC tests.